addons/cargo/functions/fnc_setSpace.sqf

```diff
@@ -35,10 +35,18 @@ if (
     {_space == _vehicle getVariable [QGVAR(space), CARGO_SPACE(typeOf _vehicle)]}
 ) exitWith {};
 
+private ["_loaded", "_cargo"];
 // Apply new space globally
 // Necessary to update value, even if no space, as API could be used again
 _vehicle setVariable [QGVAR(hasCargo), _space > 0, true];
-_vehicle setVariable [QGVAR(space), _space, true];
+//Account for cargo already in the vehicle
+_loaded = _vehicle getVariable [QGVAR(loaded), []];
+_cargo = 0;
+{
+  _cargo += _x call FUNC(getSizeItem)
```

Comment:
This wasn't tested at all, was it?

Also 4 spaces indentation, read coding guidelines.

Reply:
I tested it by repeating the steps in #7131 and confirming the issue is resolved in that case.
Am I missing something?

Reply:
How? SQF doesn't have `+=`.

Reply:
...oops
That will be because my test case is bad (I'm setting the value to what it _should_ already be, but if it errors then nothing happens so the problem doesn't arise).
I'll fix it and test more thoroughly. Not sure why the error didn't pop up...

Reply:
Should be fixed now